show/muxcable.py

```diff
@@ -336,3 +338,39 @@ def config(port, json_output):
             click.echo(tabulate(print_data, headers=headers))
 
         sys.exit(CONFIG_SUCCESSFUL)
+
+
+@muxcable.command()
+@click.argument('port', required=True, default=None, type=click.INT)
+@click.argument('target', required=True, default=None, type=click.INT)
+def berinfo(port, target):
+
+    if os.geteuid() != 0:
+        click.echo("Root privileges are required for this operation")
+        sys.exit(CONFIG_FAIL)
```

Comment:
`CONFIG_SUCCESSFUL` / `CONFIG_FAIL` names don't really apply to the show command, as nothing is being configured. Can we change them to `EXIT_SUCCESS` and `EXIT_FAILURE`, or similar? And what is the difference between the `CONFIG_*` values and the `STATUS_*` codes?

Reply:
fixed. changed these to `EXIT_FAILURE` . Actually all these error codes require rework now that I read your comment. Originally i wrote `STATUS_ ` and `CONFIG_` to differentiate between 
`show muxcable status` and
`show muxcable config` but they are confusing as we are not configuring everything in config. But hwproxy now already has the error codes handed over. Maybe just rename them and keep the codes(integer value) as is ?